set-height-width-blazor-reportviewer.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to Set the Height and Width of the Blazor Report Viewer
+description: How to set the position, height, and width of the Blazor Report Viewer
+type: how-to
+page_title: Setting the Height and Width of the Blazor Report Viewer
+slug: set-height-width-blazor-reportviewer
+position: 
+tags: 
+ticketid: 1479639
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>14.1.20.618</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+How to set the height and width of the ReportViewer component?
```

Comment:
Maybe you can say "This KB article lists the required steps for setting the height ...".  I am not sure if using a question is really appropriate for a description.

Reply:
I will change this to:
```suggestion
This article describes how to set the height and width of the ReportViewer component.
```
I am trying to keep the title as close to the initial query of the client as possible.